pycomlink/processing/wet_dry/nearby_wetdry.py

```diff
@@ -76,126 +77,84 @@ def calc_distance_between_cml_endpoints(
     # which defines nearby CMLs is larger than the distance between endpoints
     # of a CML (aka its length). Otherwise, the signal levels of CMLs with
     # length > r will not be considered in the nearby approach
-    ds = xr.where(
-        ds.cml_id1 == ds.cml_id2,
-        0,
-        ds)
+    ds = xr.where(ds.cml_id1 == ds.cml_id2, 0, ds)
     return ds
 
 
-def instantaneous_to_minmax_data(rsl, tsl, length, interval=15, timeperiod=24, min_hours=6):
+def nearby_wetdry(
+    pmin,
+    ds_dist,
+    radius=15,
+    thresh_median_P=-1.4,
+    thresh_median_PL=-0.7,
+    min_links=3,
+    interval=15,
+    timeperiod=24,
+    min_hours=6,
+):
     """
-    calculating pmin from instanteanousy measured rsl and tsl values
+    Classification of rainy and dry periods from diagnostic (min-max) CML signal
+    levels following the nearby link approach.
```

Comment:
State here that this is based on [1] and that the default values and variable names are also taken from there.

Reply:
Docstring updated to:
```
    Classification of rainy and dry periods from diagnostic (min-max) CML signal
    levels following the nearby link approach from Overeem et al. (2016).
    Variable names are adopted from them.
```